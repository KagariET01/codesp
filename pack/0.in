10 816
503 703 271 593 328 243 414 555 708 663
