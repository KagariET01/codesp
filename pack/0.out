[H[2J[3Jbuild successful
630
-181 656
-282 454
-289 440
-290 438
-33 953
-23 974
-22 988
-25 998
-29 999
-44 1000
-50 1000
-65 1000
-82 1000
-90 1000
-98 1000
-109 1000
-128 1000
-145 1000
-152 1000
-156 1000
-171 1000
-181 1000
-182 1000
-197 1000
-201 998
-207 994
-210 990
-209 997
-211 998
-215 1000
-265 1000
-277 1000
-338 1000
-339 1000
-340 1000
-345 1000
-354 1000
-364 1000
-384 1000
-392 1000
-397 1000
-424 1000
-448 1000
-457 1000
-487 1000
-503 1000
-529 1000
-541 1000
-547 1000
-548 1000
-568 1000
-573 1000
-583 1000
-612 1000
-623 1000
-627 1000
-629 1000
-683 1000
-701 1000
-715 1000
-732 1000
-753 1000
-780 1000
-785 1000
-827 1000
-844 1000
-864 1000
-866 1000
-891 1000
-908 1000
-911 1000
-918 1000
-925 1000
-927 1000
-928 1000
-947 1000
-966 1000
-977 1000
-978 944
-1000 -901
-982 598
-979 863
-978 991
-980 998
-982 994
-983 990
-984 895
-1000 -876
-989 312
-984 903
-984 927
-984 929
-984 930
-984 932
-985 979
-987 998
-988 912
-990 622
-1000 -852
-992 303
-988 977
-988 1000
-1000 -839
-993 186
-990 688
-989 868
-989 873
-989 881
-989 909
-989 946
-989 952
-989 960
-989 983
-990 990
-991 996
-993 958
-1000 -760
-994 688
-993 975
-993 980
-995 1000
-996 748
-1000 -678
-997 341
-996 783
-996 796
-996 824
-996 830
-996 868
-996 872
-996 926
-996 953
-997 669
-998 259
-1000 -617
-999 -248
-997 701
-997 705
-997 714
-997 752
-997 763
-997 785
-997 787
-997 838
-997 862
-997 918
-997 976
-997 995
-1000 -524
-998 428
-998 482
-998 489
-998 529
-998 559
-998 583
-998 618
-998 641
-998 662
-998 677
-998 682
-998 693
-998 700
-998 710
-998 774
-998 804
-998 827
-1000 -454
-999 75
-998 913
-998 1000
-1000 -352
-999 299
-999 313
-999 338
-999 349
-999 360
-999 390
-999 392
-999 406
-999 481
-999 485
-999 488
-999 571
-999 602
-999 615
-999 717
-999 726
-999 733
-999 738
-999 743
-999 745
-999 756
-999 762
-999 795
-999 837
-999 850
-999 880
-999 945
-1000 997
-1000 -983
-997 -997
-987 -999
-975 -1000
-956 -1000
-953 -1000
-947 -1000
-942 -1000
-890 -1000
-888 -1000
-844 -1000
-771 -1000
-750 -1000
-726 -1000
-717 -1000
-711 -1000
-693 -1000
-675 -1000
-648 -1000
-632 -1000
-625 -1000
-598 -1000
-554 -1000
-520 -1000
-518 -1000
-506 -1000
-493 -1000
-459 -1000
-456 -1000
-451 -1000
-439 -1000
-436 -1000
-399 -1000
-396 -1000
-395 -1000
-392 -1000
-373 -1000
-365 -1000
-354 -1000
-343 -1000
-300 -1000
-282 -1000
-236 -1000
-195 -1000
-183 -1000
-179 -1000
-178 -1000
-172 -1000
-170 -1000
-163 -1000
-118 -1000
-85 -1000
-19 -1000
-17 -1000
-14 -1000
1 -1000
20 -1000
35 -1000
87 -1000
93 -1000
132 -1000
141 -1000
150 -1000
155 -1000
180 -1000
187 -1000
188 -1000
235 -1000
242 -1000
288 -1000
321 -1000
341 -1000
342 -1000
355 -1000
358 -1000
372 -1000
396 -1000
439 -1000
457 -1000
462 -1000
468 -1000
488 -1000
523 -1000
561 -1000
598 -1000
601 -1000
602 -1000
615 -1000
640 -1000
653 -1000
675 -1000
710 -1000
724 -1000
731 -1000
736 -1000
747 -1000
768 -1000
779 -1000
797 -1000
819 -1000
821 -1000
823 -1000
837 -1000
855 -1000
864 -1000
866 -1000
879 -1000
902 -1000
915 -1000
925 -1000
926 -1000
978 -1000
995 -999
1000 -995
1000 -984
992 -983
978 -982
936 -982
888 -982
874 -982
872 -982
869 -982
856 -982
837 -982
828 -982
823 -982
802 -982
785 -982
766 -982
742 -982
727 -982
712 -982
687 -982
644 -982
630 -982
588 -982
579 -982
568 -982
533 -982
521 -982
492 -982
443 -982
425 -982
420 -982
415 -982
398 -982
379 -982
969 -981
957 -981
920 -981
905 -981
864 -981
824 -981
795 -981
766 -981
679 -981
627 -981
613 -981
612 -981
980 -980
963 -980
962 -980
945 -980
941 -980
922 -980
892 -980
867 -980
827 -980
823 -980
822 -980
982 -979
972 -978
971 -978
940 -978
915 -978
988 -977
978 -977
961 -977
952 -977
990 -976
991 -974
998 -957
1000 -948
1000 -944
1000 -930
1000 -926
1000 -923
1000 -921
1000 -900
1000 -834
1000 -804
1000 -786
1000 -784
1000 -752
1000 -740
1000 -705
1000 -693
1000 -678
1000 -669
1000 -662
1000 -661
1000 -657
1000 -647
1000 -630
1000 -619
1000 -616
1000 -599
1000 -554
1000 -499
999 -483
995 -484
999 -476
1000 -471
1000 -449
1000 -447
1000 -410
1000 -404
1000 -402
1000 -388
1000 -378
1000 -376
1000 -357
1000 -354
1000 -349
1000 -345
997 -325
1000 -310
1000 -309
1000 -307
1000 -281
1000 -201
1000 -182
1000 -160
1000 -112
1000 -91
1000 -88
1000 -36
1000 -27
1000 -12
1000 -5
1000 -4
1000 7
995 13
994 13
990 12
957 -4
943 -11
899 -33
847 -59
839 -63
777 -94
761 -102
745 -110
681 -142
942 -11
989 14
999 25
1000 59
1000 64
1000 71
1000 75
1000 85
1000 96
1000 126
1000 151
1000 190
1000 193
1000 207
1000 226
1000 292
1000 328
1000 336
1000 345
1000 377
1000 419
1000 456
1000 457
1000 464
1000 474
1000 510
998 515
1000 556
1000 563
1000 566
1000 570
1000 580
1000 606
1000 610
1000 660
1000 670
1000 685
1000 693
1000 709
1000 713
1000 741
1000 745
1000 755
1000 797
1000 806
1000 807
1000 817
1000 820
1000 856
1000 911
1000 913
1000 959
999 999
984 998
978 996
971 989
964 982
947 965
919 937
900 918
877 895
864 882
821 839
802 820
795 813
740 758
735 753
726 744
720 738
719 737
699 717
622 640
610 628
606 624
605 623
602 620
959 978
901 920
887 906
877 896
875 894
867 886
817 836
801 820
784 803
736 755
720 739
719 738
708 727
696 715
980 1000
969 989
967 987
952 972
942 962
932 952
928 948
925 945
919 939
908 928
889 909
886 906
873 893
840 860
836 856
959 980
916 937
857 878
852 873
949 971
959 983
963 988
970 997
950 1000
922 1000
882 1000
856 1000
852 1000
846 1000
833 1000
818 1000
768 1000
752 1000
743 1000
721 1000
684 1000
681 1000
678 1000
671 1000
631 1000
624 1000
623 1000
576 1000
574 1000
501 1000
496 1000
487 1000
482 1000
465 1000
461 1000
435 1000
428 1000
426 1000
417 1000
365 1000
326 1000
314 1000
235 1000
232 1000
219 1000
214 1000
203 1000
202 1000
193 1000
186 1000
171 1000
160 1000
159 1000
158 1000
124 1000
112 1000
95 1000
73 1000
38 1000
25 1000
19 1000
15 1000
4 1000
-5 998
-11 990
-15 984
-28 962
-90 838
-119 780
-120 778
-122 774
-135 748
