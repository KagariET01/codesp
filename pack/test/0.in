7 3
abcdefg
abc
1 13

