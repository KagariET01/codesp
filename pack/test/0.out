[['hello', 'world!'], ['what', 'is', 'the', 'weather', 'likke', 'out', 'there?'], ['are', 'you', 'ok?'], ["wht's", "today's", 'date?'], ['excuse', 'me!']]
Hello World! 
What Is The Weather Likke Out There? 
Are You Ok? 
Wht's Today's Date? 
Excuse Me! 
